tests/ttnn/unit_tests/gtests/test_multi_device.cpp

```diff
@@ -45,4 +45,17 @@ TEST_F(T3kMultiDeviceFixture, TestGetDistributedTensorConfigFromMultiDeviceStora
     EXPECT_TRUE(std::holds_alternative<ReplicateTensor>(distributed_tensor_config));
 }
 
+Test_F(T3kMultiDeviceFixture, TestRingAPI) {
```

Comment:
Where should I put testing for ring api?

Reply:
That looks like a fine place.